spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -699,6 +699,21 @@ public void setConsumersPerQueue(Integer consumersPerQueue) {
 		 */
 		private Long replyTimeout;
 
+		/**
+		 * The name of the default exchange to use for send operations.
+		 */
+		private String exchange;
```

Comment:
The default exchange is `""` so it would be nice if the property would say that rather than `null`